perfil/mandate/management/commands/load_deputy_info.py

```diff
@@ -36,6 +36,7 @@ def serialize(self, reader, total, progress_bar):
             row['name'] = row['name'].replace('  ', ' ')
             person_id = self.cache.get(person_keys_birthdate(row))
             if person_id:
+                break
```

Comment:
Was this `break` intentionally left here?

Reply:
this should totally not be here o.O